### ISSUE TITLE
Crash on Windows

### ISSUE THREAD

**Issue by janoszen** [exoscale/python-exoscale#22]
The Python library crashes on Windows because `__init__.py` relies on the `HOME` environment variable to be set in like 21. Instead, pathlib should be used:

```
from pathlib import Path
_DEFAULT_CONFIG_FILE = os.path.join(str(Path.home()), ".exoscale", "config.toml")
```

**Comment by Patralos** [comment]
+1

**Comment by pyr** [comment]
Thanks for the report, will address shortly

**Comment by falzm** [comment]
Fixed in 0.5.0.
